WordPress/Classes/Networking/PlansRemote.swift

```diff
@@ -208,8 +209,14 @@ private func mapPlanFeaturesResponse(response: AnyObject) throws -> PlanFeatures
                     let planSpecificDescription = planInfo["description"] {
                         description = planSpecificDescription
                 }
+                    
+                // The slug currently returned for 'no ads' includes the text '/no-adverts.php'
+                // Remove this to 'normalize' all of the slugs
+                if let index = slug.characters.indexOf("/") {
```

Comment:
No plans at the moment – afaik it would be a fair amount of effort as (I'm told) the frontend matches on slugs in various places.